src/control/ControlUtils.cpp

```diff
@@ -219,29 +92,46 @@ Vector2 ControlUtils::accelerationLimiter(const Vector2 &targetVel, const Vector
     return prevVel + deltaVel.stretchToLength(finalAcceleration);
 }
 
-/// Get the intersection of two lines
-Vector2 ControlUtils::twoLineIntersection(const Vector2 &a1, const Vector2 &a2, const Vector2 &b1, const Vector2 &b2) {
-    // https://en.wikipedia.org/wiki/Line%E2%80%93line_intersection
-    double denominator = ((a1.x - a2.x) * (b1.y - b2.y) - (a1.y - a2.y) * (b1.x - b2.x));
-    if (denominator != 0) {
-        double numerator = ((a1.x - b1.x) * (b1.y - b2.y) - (a1.y - b1.y) * (b1.x - b2.x));
-        double t = numerator / denominator;
-        return (a1 + (Vector2){t, t} * (a2 - a1));
-    } else {
-        return Vector2();
+/// Calculate the force of a given vector + a certain type.
+/// the basic formula is: force = weight/distance^2 * unit vector
+Vector2 ControlUtils::calculateForce(const Vector2 &vector, double weight, double minDistance) {
+    // if the object is close enough, it's forces should affect. Otherwise don't change anything.
```

Comment:
affect what?

Reply:
That documentation was not added by me, some parts of the code have been moved to another location...